components/email-mgt/org.wso2.carbon.email.mgt/src/main/java/org/wso2/carbon/email/mgt/util/I18nEmailUtil.java

```diff
@@ -271,5 +273,27 @@ public static Collection createTemplateType(String normalizedTemplateName, Strin
         return collection;
     }
 
+    /**
+     * Prepend the operation scenario to the existing exception error code.
+     * (Eg: USR-20045)
+     *
+     * @param exceptionErrorCode Existing error code.
+     * @param scenario           Operation scenario
+     * @return New error code with the scenario prepended (NOTE: Return an empty String if the provided error code is
```

Comment:
Put the full stop, end of the comment

Reply:
`.` has been added to the method level comment. We do not add `.` at the end of param descriptions.